object_templates/fact_spec.erb

```diff
@@ -7,9 +7,42 @@ require 'facter/<%= name %>'
 describe <%= name.to_sym.inspect %>, type: :fact do
   subject(:fact) { Facter.fact(<%= name.to_sym.inspect %>) }
 
+  # Mocking Facts
+  # You will most likely need to mock other facts if your custom fact relies up on them
+  # You can mock a existing core fact via:
+  #  allow(Facter.fact(:fqdn)).to receive(:value).and_return('test.example.com')
+  #  allow(Facter).to receive(:value).with(:fqdn).and_return('test.example.com')
+  # This all depends on how you utilize it in the fact code
+
+  #
```

Comment:
Shall we remove this line along with the comment?

Reply:
updated